Shared/Samples/Create convex hull around points/CreateConvexHullAroundPointsView.swift

```diff
@@ -0,0 +1,130 @@
+                        resetIsDisabled = true
+                    }
+                    .disabled(resetIsDisabled)
+                }
+            }
+    }
+}
+
+private extension CreateConvexHullAroundPointsView {
+    /// The view model for this sample.
+    private class Model: ObservableObject {
+        /// A map with a topographic basemap.
+        let map: Map
+        
+        /// An array of input points to be used in creating the convexHull.
+        var inputPoints: [Point] = []
+        
+        /// An array that contains the graphics overlays for the sample.
+        var graphicsOverlays: [GraphicsOverlay] {
+            return [pointsGraphicsOverlay, convexHullGraphicsOverlay]
+        }
+        
+        /// The graphics overlay for the input points graphics.
+        var pointsGraphicsOverlay: GraphicsOverlay
+        
+        /// The graphics overlay for the convex hull graphic.
+        private let convexHullGraphicsOverlay: GraphicsOverlay
+        
+        /// A red simple marker symbol to display where the user tapped on the map.
+        private let markerSymbol: SimpleMarkerSymbol
+        
+        /// A blue simple line symbol for the outline of the convex hull graphic.
+        private let lineSymbol: SimpleLineSymbol
+        
+        /// A hollow polygon simple fill symbol for the convex hull graphic.
+        private var fillSymbol: SimpleFillSymbol
+        
+        init() {
+            map = Map(basemapStyle: .arcGISTopographic)
+            
+            markerSymbol = SimpleMarkerSymbol(style: .circle, color: .red, size: 10)
+            lineSymbol = SimpleLineSymbol(style: .solid, color: .blue, width: 4)
+            fillSymbol = SimpleFillSymbol(style: .noFill, outline: lineSymbol)
+            
+            pointsGraphicsOverlay = GraphicsOverlay()
+            pointsGraphicsOverlay.renderer = SimpleRenderer(symbol: markerSymbol)
+            convexHullGraphicsOverlay = GraphicsOverlay()
+        }
+        
+        /// Reset the points and graphics.
+        func reset() {
+            inputPoints.removeAll()
+            pointsGraphicsOverlay.removeAllGraphics()
+            convexHullGraphicsOverlay.removeAllGraphics()
+        }
+        
+        /// Create the convex hull graphic using the inputPoints.
+        func createConvexHull() {
+            // Normalize points and create convex hull geometry.
+            if let normalizedPoints = GeometryEngine.normalizeCentralMeridian(of: Multipoint(points: inputPoints)),
+               let convexHullGeometry = GeometryEngine.convexHull(for: normalizedPoints) {
+                // Set the symbol depending on the geometry type of the convex hull.
+                let symbol: Symbol?
+                switch convexHullGeometry {
+                case is Point:
+                    symbol = markerSymbol
+                case is Polyline:
+                    symbol = lineSymbol
+                case is Polygon:
```

Comment:
```suggestion
                case is ArcGIS.Polygon:
```
To resolve this warning:

<img width="598" alt="Screenshot 2023-06-02 at 10 26 58 AM" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/ef0fa98d-b2db-45bd-a6b9-fed18512972d">